geps/gep-1713/index.md

```diff
@@ -438,35 +446,38 @@ spec:
     sectionName: second
 ```
 
-For instance, the following `HTTPRoute` attempts to attach to a listener defined in the parent `Gateway` using the sectionName `foo`. This is not valid and the route's status `Accepted` condition should be set to `False`
-
+To attach to listeners in both a `Gateway` and `ListenerSet` the route MUST have two `parentRefs`:
 ```yaml
 apiVersion: gateway.networking.k8s.io/v1
 kind: HTTPRoute
 metadata:
   name: httproute-example
 spec:
   parentRefs:
-  - name: some-workload-listeners
+  - name: second-workload-listeners
     kind: ListenerSet
+    sectionName: second
+  - name: parent-gateway
+    kind: Gateway
     sectionName: foo
 ```
 
-To attach to listeners in both a `Gateway` and `ListenerSet` the route MUST have two `parentRefs`:
+For instance, the following `HTTPRoute` attempts to attach to a listener defined in the parent `Gateway` using the sectionName `foo`. This is not valid and the route's status `Accepted` condition should be set to `False`
+
 ```yaml
 apiVersion: gateway.networking.k8s.io/v1
 kind: HTTPRoute
 metadata:
   name: httproute-example
 spec:
   parentRefs:
-  - name: second-workload-listeners
-    kind: ListenerSet
-    sectionName: second
-  - name: parent-gateway
+  - name: some-workload-listeners
     kind: Gateway
     sectionName: foo
 ```
+>--- Ricardo - above is a bit confusing, maybe ask Dave some clarification
```

Comment:
@dprotaso the example above is a bit confusing for me, probably because the API changed from when it was implemented.

I was wondering what it means for an HTTPRoute to try to attach a listener defined on a Gateway, using a section name `foo`. If this section name exists on the Gateway it is valid, right? eg.:

```
  listeners:
  - name: foo
    port: 80
    protocol: HTTP
```

So how can the situation above be invalid? Maybe we can get the full yaml definition, with Gateway, ListenerSet and HTTPRoute for clarification here?

Thanks!